spec/unit/azure_server_create_spec.rb

```diff
@@ -759,7 +759,12 @@ def test_params(testxml, chef_config, role_name, host_name)
         expect(@server_instance).to receive(:get_chef_extension_private_params)
         expect(@server_instance).to receive(:is_image_windows?).exactly(4).times.and_return(true)
         expect(@server_instance).to receive(:wait_until_virtual_machine_ready).exactly(1).times.and_return(true)
-       @server_instance.run
+        @server_instance.run        
+        testxml = Nokogiri::XML(@receivedXML)        
+        testxml.css('InputEndpoint Protocol:contains("TCP")').each do | port |          
+          expect(port.parent.css("LocalPort").text).to_not eq("5985")
+          expect(port.parent.css("LocalPort").text).to_not eq("22")
```

Comment:
Can we add this assertion for Linux instance create and bootstrap_protocol = cloud_api ?